html-forms.tar/html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <p>Email</p>
+    <input type="text" id="comment" name="comment" value="" />
+    <p>Why is coding cool?</p>
+    <select id="reason" name="userchoice">
+      <option value="It is fun">It is fun</option>
+      <option value="It improves your logical skill">It improves your logical skill</option>
+      <option value="It helps you be more adaptable to technology">It helps you be more adaptable to technology</option>
```

Comment:
It'll probably be better to make these `value`s equal to single word values (it get's tricky trying to use full sentence strings).